packages/contracts/contracts/token/Sale.sol

```diff
@@ -10,6 +10,7 @@ import {ReentrancyGuard} from "@openzeppelin/contracts/security/ReentrancyGuard.
 import {ISale} from "./ISale.sol";
 import {RisingTide} from "../RisingTide/RisingTide.sol";
 import {FractalRegistry} from "../fractal_registry/FractalRegistry.sol";
+import {Math} from "../libraries/Math.sol";
 
 import "hardhat/console.sol";
```

Comment:
We can probably remove this import, right? Or do we need to print something in this contract?